ibc-core/ics04-channel/types/src/error.rs

```diff
@@ -18,13 +18,13 @@ use crate::Version;
 /// Errors that arise from the ICS04 Channel module
 #[derive(Debug, Display, derive_more::From)]
 pub enum ChannelError {
```

Comment:
The same comment about backticks.

Reply:
Explained in the [ADR](https://github.com/cosmos/ibc-rs/pull/1344)

Reply:
Awesome. Thanks for including that.